X/XML2/build_tarballs.jl

```diff
@@ -3,13 +3,13 @@
 using BinaryBuilder
 
 name = "XML2"
-ygg_version = v"2.13.6"
-version = v"2.13.5" # Building version 2.13.5; bumped to *.6 to build for riscv
+ygg_version = v"2.13.7"
+version = v"2.13.6" # Bumped by 0.0.1 when we built for riscv
```

Comment:
Why not follow upstream?

Reply:
I don't know when one can follow upstream and when one must increase the version number.

I'll do that, then.